tests/monotouch-test/Compression/CompressionStreamTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+
+using Compression;
+using Foundation;
+
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.Compression {
+
+	[TestFixture]
+	public class CompressionStreamTest {
+
+#if MONOMAC
+                string lzeCompressedFilePath = Path.Combine (NSBundle.MainBundle.BundlePath, "Contents/Resources/compressed_lze");
+                string lzeUncompressedFilePath = Path.Combine (NSBundle.MainBundle.BundlePath, "Contents/Resources/uncompressed.json");
+                
+#else
+                string lzeCompressedFilePath = Path.Combine (NSBundle.MainBundle.BundlePath, "compressed_lze");
+                string lzeUncompressedFilePath = Path.Combine (NSBundle.MainBundle.BundlePath, "uncompressed.json");
+
+#endif
+
+		string tempPath;
+		[Test]
+		public async Task TestDecode ()
+		{
+		
+			using (var inputStream = File.OpenRead (lzeCompressedFilePath))
+			using (var outputStream = File.Create (tempPath))
```

Comment:
ArgumentNullException: `tempPath` is never initialized.

Reply:
it's in `SetUp`